src/index.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+const state = {
+  tempValue: 75,
+  color: 'orange',
+  landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  skyscape: '',
+  background: "url('../assets/summer.jpg')",
+  city: 'Orlando',
+  lat: '',
+  lon: '',
+};
+
+// API calls
+const getLatLon = () => {
+  const inputCityElement = document.getElementById('inputCity').value;
+  axios
+    .get('http://127.0.0.1:5000/location', { params: { q: state.city } })
```

Comment:
Consider using a constant variable like LOCATION_URL instead of string literal here.

"Constants free the programmer from having to remember what each literal should be. Often values that stay constant throughout the program have a business meaning. If there are several such values, the programmer can define them all in the beginning of the program and then work with the easier-to-remember constant names." From: https://www.diffen.com/difference/Constant_vs_Literal